stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -134,15 +134,22 @@ spec:
               protocol: TCP
             {{- end }}
           {{- end }}
+          {{- $useHostPort := .Values.controller.daemonset.useHostPort -}}
           {{- range $key, $value := .Values.tcp }}
             - name: "{{ $key }}-tcp"
               containerPort: {{ $key }}
               protocol: TCP
+              {{- if $useHostPort }}
```

Comment:
Lets match the if statements above instead of putting the value into a temporary var.
```
{{- if .Values.controller.daemonset.useHostPort }}
```